include/seqan3/range/views/kmer_hash.hpp

```diff
@@ -90,7 +90,7 @@ class kmer_hash_view : public std::ranges::view_interface<kmer_hash_view<urng_t>
          * \{
          */
         //!\brief Type for distances between iterators.
-        using difference_type = typename it_t::difference_type;
+        using difference_type = typename std::iterator_traits<it_t>::difference_type;
```

Comment:
```suggestion
        using difference_type = std::iter_difference_t<it_t>;
```

C++20-style traits should be used for iterators and ranges everywhere now.

Reply:
We should write that down in the wiki

Reply:
that would be great if you could do that.

Reply:
I added a card in the documentation project since the wiki is outdated.